backend/migrator/migration/LATEST.sql

```diff
@@ -417,7 +417,8 @@ CREATE TABLE worksheet_organizer (
     id serial PRIMARY KEY,
     worksheet_id integer NOT NULL REFERENCES worksheet(id) ON DELETE CASCADE,
     principal_id integer NOT NULL REFERENCES principal(id),
-    starred boolean NOT NULL DEFAULT false
+    starred boolean NOT NULL DEFAULT false,
+    category text NOT NULL DEFAULT ''
```

Comment:
Do we need any index for listing all categories?

Query: list all sub-folders within a given folder?

Reply:
Good question, I'm not sure if we need this feature, but we can prepare the index for this